platform/macos/docs/guides/For Style Authors.md

```diff
@@ -243,6 +244,37 @@ In style JSON | In Objective-C | In Swift
 `fill-translate` | `MGLFillStyleLayer.fillTranslation` | `MGLFillStyleLayer.fillTranslation`
 `fill-translate-anchor` | `MGLFillStyleLayer.fillTranslationAnchor` | `MGLFillStyleLayer.fillTranslationAnchor`
 
+### FillExtrusion style functions
```

Comment:
This section will go away once #8526 lands in master as part of #8672. If that PR lands first, you can rerun `make darwin-style-code` to eliminate it; otherwise, @boundsj can take care of that in #8672.